macros/plotSCurveFitResults.py

```diff
@@ -383,9 +394,19 @@
 
         canvScurveEffPed_DetSum.cd()
         if idx == 0:
-            dict_ScurveEffPed[chamberAndScanDatePair][-1].Draw("E1")
+	    dict_ScurveEffPed[chamberAndScanDatePair][-1].Draw("E1")
         else:
             dict_ScurveEffPed[chamberAndScanDatePair][-1].Draw("sameE1")
+	
+	canvScurveSigma_boxPlot.cd()
+	dict_ScurveSigma_boxPlot[chamberAndScanDatePair].SetFillColorAlpha(getCyclicColor(idx), 0.3)
+	dict_ScurveSigma_boxPlot[chamberAndScanDatePair].SetLineColor(getCyclicColor(idx))
+	if idx == 0:
+	    dict_ScurveSigma_boxPlot[chamberAndScanDatePair].GetXaxis().SetTitle("VFAT position")
+            dict_ScurveSigma_boxPlot[chamberAndScanDatePair].GetYaxis().SetTitle("Noise (fC)")
+	    dict_ScurveSigma_boxPlot[chamberAndScanDatePair].Draw("candle1")
```

Comment:
Again here, your editor probably is using whitespace that does not match.  If you where to use `vim` the relevant line in your `$HOME/.vimrc` to convert tab's to spaces is:

```
filetype plugin indent on
" show existing tab with 4 spaces width
set tabstop=4                                                                                                                                                                                                                                
" when indenting with '>', use 4 spaces width
set shiftwidth=4
" On pressing tab, insert 4 spaces
set expandtab
```